cwl/src/main/scala/cwl/CommandLineTool.scala

```diff
@@ -74,23 +75,49 @@ case class CommandLineTool private(
   }
 
   def buildTaskDefinition(parentWorkflow: Option[Workflow], validator: RequirementsValidator): ErrorOr[CallableTaskDefinition] = {
+    import wom.RuntimeAttributesKeys._
+    
+    def processRequirement(requirement: Requirement): Map[String, WomExpression] = {
```

Comment:
ToL IMO would be nice to tuck this away either as a Poly1/fold or perhaps a private def.